docs/source/asr/speaker_diarization/api.rst

```diff
@@ -6,12 +6,11 @@ Model Classes
 -------------
 .. autoclass:: nemo.collections.asr.models.ClusteringDiarizer
     :show-inheritance:
-    :members: init_speaker_model, set_vad_model, init_vad_model, setup_vad_test_data, setup_spkr_test_data, run_vad, extract_embeddings, path2audio_files_to_manifest, diarize, make_nemo_file_from_folder, save_to, unpack_nemo_file, restore_from
-
+    :members: 
```

Comment:
What is the temp workaround for the API part?

Reply:
I just did not include diarization_utils.py and decode_timestamp_utils.py. The docstrings in these files should be heavily edited to render "autoclass" properly. BTW, these variables are removed. Not needed.

Reply:
Maybe we need to find a time to discuss docstring and api part for doc. Let's keep it now.